force-app/main/default/aura/modalWindows/expressionsEvent/expressionsEvent.evt

```diff
@@ -0,0 +1,4 @@
+<aura:event type="APPLICATION" description="Event template">
```

Comment:
name needs to be clearer. what triggers this event and what purpose does it server?